crates/wasm/src/wasm_planner.rs

```diff
@@ -27,10 +28,10 @@ pub struct WasmPlanner {
 #[wasm_bindgen]
 impl WasmPlanner {
     #[wasm_bindgen(constructor)]
-    pub async fn new() -> Result<WasmPlanner, Error> {
+    pub async fn new(idb_constants: IndexedDbConstants) -> Result<WasmPlanner, Error> {
         let planner = WasmPlanner {
             planner: Planner::new(OsRng),
-            storage: IndexedDBStorage::new().await?,
+            storage: IndexedDBStorage::new(idb_constants).await?,
```

Comment:
It looks like the constructor now requires opening up an IndexedDb connection. Can you add a code comment about this fact?

Or, do we need to opening up that connection when we instantiate the class? Could we not do it when we make the request? Just a thought.

Reply:
We can choose not to open IndexedDb connection in constructor, but then we will have to open IndexedDb connection on every request
Added documentation

Reply:
Yeah, that's fair